src/terra-dev-site/doc/application/how-to.2/use-translated-strings.doc.mdx

```diff
@@ -1,25 +1,26 @@
 # How To Use Translated Strings
 
-Translated strings can be retrieved using the `intl` object provided by `injectIntl` HOC-generator provided by [react-intl](https://github.com/formatjs/react-intl/blob/master/docs/API.md#injectintl).
+Translated strings can be retrieved using the [ApplicationIntlContext](/components/terra-application/application/reference/contexts/application-intl-context). An `ApplicationIntlContext.Provider` is configured and provided by [ApplicationBase](/components/terra-application/application/reference/components/application-base). The strings will be translated based upon the `locale` prop provided to ApplicationBase.
```

Comment:
The **pre-aggregated translation** strings will be ~translated~ **loaded** based upon the `locale` prop provided to ApplicationBase.


It'd be worth mention terra-i18n's loading strategy and the fallback strategy to english, even if it's linking to the docs.